website/docs/product/telemetry.md

```diff
@@ -1,65 +1,113 @@
+---
+description: This page provides information on the data collected by Appsmith.
+---
+
 # Telemetry
 
 Telemetry in Appsmith refers to the collection of data about how users interact with the platform. This data helps the Appsmith team identify usage patterns, troubleshoot issues, and make informed decisions about new features and improvements.
 
 ## Data collected by Appsmith
 
-Appsmith is a lightweight proxy and **does not capture** any data returned by your APIs, databases, or third-party tools. All data captured from a self-hosted instance is completely anonymised.
+Appsmith is a lightweight proxy and **does not capture** any data returned by your APIs, databases, or third-party tools. All data captured from a self-hosted instance is completely anonymised for user information.
```

Comment:
"Any user data captured from a self-hosted instance is completely anonymised"

Reply:
"All data captured from a self-hosted instance is completely anonymised for user information" explicitly emphasizes that the anonymization is specifically for user information. Let's keep it as is.